images/ci-prune-buildcache/entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+if [ -z $1 ]; then
+  now=$(date --iso-8601)
+else
+  now=$1
+fi
+
+snapshot_dir="$PWD/snapshot-${now}"
+
+# Use the $SPACKROOT/etc/ dir for configs
+SPACK_DISABLE_LOCAL_CONFIG=1
```

Comment:
Is this leftover from debugging?

Reply:
nope, this is intentional. Unless you are running spack with a specific config root or on a local system this should probably be set. It prevents accidentally injecting persistent spack configs for detecting spack configs used by another system installation of spack.

Reply:
Oh, then does it need to be exported?

Reply:
it probably should be to make it clear. But I think it was fine since spack runs in the current shell, it worked for me locally.

Reply:
My lack of bash-fu knowledge might be showing, but aren't only exported variables passed to the environment of child processes? I don't see how spack could ever see this value.